src/main/java/util/LottoNumberGenerator.java

```diff
@@ -1,33 +1,35 @@
 package util;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
+import domain.LottoNumber;
+
+import java.util.*;
 
 public class LottoNumberGenerator {
 
-    private static List<Integer> allNumbers;
     public static final Integer MIN_LOTTO_NUMBER = Integer.valueOf(1);
     public static final Integer MAX_LOTTO_NUMBER = Integer.valueOf(45);
```

Comment:
```suggestion
    public static final Integer MIN_LOTTO_NUMBER =1
    public static final Integer MAX_LOTTO_NUMBER =45
```
그냥 이렇게 작성하셔도 됩니다! ㅎㅎ